src/equations/laplace_diffusion_2d.jl

```diff
@@ -21,6 +21,10 @@ function flux(u, grad_u, equations::LaplaceDiffusion2D)
   return SVector(equations.diffusivity * dudx, equations.diffusivity * dudy)
 end
 
+function penalty(u_outer, u_inner, inv_h, equations::LaplaceDiffusion2D, dg::LDG)
+  return dg.penalty_parameter * (u_outer - u_inner) * equations.diffusivity * inv_h
+end
```

Comment:
Could you please add to our list of things to discuss where to put this stuff? Usually, we try to keep the `equations` as basic as possible and add specializations of the solver/equations to the solver folders.

Reply:
Is this function even used currently? Also, where is `penalty_parameter` defined? Further, I would consider passing the penalty parameter as a separate argument, such that it does not force the internal structure of the used solver. Also, is there a clear meaning of `u_outer` and `u_inner`? Finally, `inv_h` is suboptimal here imho, as it seems to be solver-specific.

Much of what I wrote, I just realized, would probably become moot if this is not part of the equations but of the solver. But this might need more discussions, I agree with @ranocha here...

Reply:
I agree that this clutters `equations` a bit, but parabolic penalty terms tend to depend very closely on the specific equation. Keeping them with the solvers might clutter things with equation-specific routines quickly.

It might make more sense to have the penalty terms live in the elixir themselves?

Reply:
Yes, could also be an option. Please add some TODO notes here and summarize the ideas in the meta-issue.